sbt-crossproject/src/main/scala/sbtcrossproject/CrossProject.scala

```diff
@@ -6,6 +6,7 @@ import scala.language.implicitConversions
 
 import sbt._
 import Keys._
+import sbt.internal.inc.ScalaInstance
```

Comment:
As its name implies, this is an internal API of sbt, so we cannot use it.

Reply:
Oops!